### ISSUE TITLE
Update ViPhOGs to remove old models that are associated with discontinued viral taxa

### ISSUE THREAD

**Issue by hoelzer** [EBI-Metagenomics/emg-viral-pipeline#104]
This needs to be done, @guille0387 detected which ViPhOGs belong to discontinued viral taxa such as the families _Siphoviridae_, _Myoviridae_, ...

Re-calculating the models is not so easy, but for now, we can simply remove these old models (which are not that many) from the ViPhOG database. 

@guille0387 I think you can provide a list of which models need to be removed. And then we can update the ViPhOG database file and the pipeline accordingly?

**Comment by mberacochea** [comment]
Alright, that sounds like a plan. Should we put a warning message for the metadata v3?.

Ping me when the v4 metadata file is ready and I'll make the required changes.

**Comment by hoelzer** [comment]
Yes, good idea. @guille0387 we could put a warning message that v4 does not include the following discontinued virus taxa (according to ICTV) anyomore and then lost them:

Siphoviridae
Podoviridae
Myoviridae
Caudovirales
Allolevivirus
Autographivirinae
Buttersvirus
Chungbukvirus
Incheonvirus
Leviviridae
Levivirus
Mandarivirus
Pbi1virus
Phicbkvirus
Radnorvirus
Sitaravirus
Vidavervirus

(Pls double-check that list)

**Comment by guille0387** [comment]
Hi Martin B,

Here’s the updated viphog metadata file (v4 as discussed in the PR).

Kind regards
,

Guillermo Rangel-Pineros
Postdoc Palaeoproteomics Group

University of Copenhagen
Faculty of Health and Medical Sciences
The Globe Institute
Øster Farimagsgade 5, bygning 7
1353 Copenhagen K
DENMARK

MOB +45 50 10 57 42
***@***.******@***.***>

On 3 Jul 2023, at 11.22, Martín Beracochea ***@***.***> wrote:



Alright, that sounds like a plan. Should we put a warning message for the metadata v3?.

Ping me when the v4 metadata file is ready and I'll make the required changes.

**Comment by guille0387** [comment]
hey @mberacochea , I just sent you an email with the updated metadata file v4

**Comment by hoelzer** [comment]
Should be solved for now with the merge of PR https://github.com/EBI-Metagenomics/emg-viral-pipeline/pull/103
